index.html

```diff
@@ -5,41 +5,242 @@
-            <p class="second-paragraph">LETS CONNECT</p>
-            <ul class="social-media">
-                <li> <a href="#"><img src="./images/github.jpg" alt="github"></a> </li>
-                <li> <a href="#"><img src="./images/medium.jpg" alt="Linkedin"></a> </li>
-                <li> <a href="#"><img src="./images/twitter.jpg" alt="medium"></a> </li>
-                <li> <a href="#"><img src="./images/Vector.jpg" alt="twitter"></a> </li>
-            </ul>
-        </div>
-    </section>
+    <div class="background-color-container">
+
+
+        <section class="first-section">
+
+            <div class="section2">
+                <h1>I'm Ekene, <br> glad to see <br> you!</h1>
+                <p class="first-paragraph">I'm a software developer! I can help you build a product , feature or website
+                    Look through
+                    some of my work and experience! If you like what you see and have a project you need coded,
+                    don't hesitate to contact me.</p>
+                <p class="second-paragraph">LETS CONNECT</p>
+                <ul class="social-media">
+                    <li> <a href="#"><img src="./images/github.jpg" alt="github"></a> </li>
+                    <li> <a href="#"><img src="./images/medium.jpg" alt="Linkedin"></a> </li>
+                    <li> <a href="#"><img src="./images/twitter.jpg" alt="medium"></a> </li>
+                    <li> <a href="#"><img src="./images/Vector.jpg" alt="twitter"></a> </li>
+                </ul>
+            </div>
+        </section>
+
+        <div class="grid-position">
+            <div class="Grid-container">
+                <section class="second-section">
+                    <img src="./images/Snapshoot-Portfolio-(3).jpg" alt="section-eader">
+                    <h1>TONIC</h1>
+
+                    <div class="to-do">
+                        <span class="can">CANOPY</span>
+                        <span class="img"><img src="./images/Counter.jpg" alt="seperation"></span>
+                        <span class="back">Back End Dev</span>
+                        <span class="img"> <img src="./images/Counter.jpg" alt="seperation"></span>
+                        <span class="num">2015</span>
+                    </div>
+                    <p class="to-do-para">A daily selection of privately personalised reads: no accounts or sign-ups
+                        required.</p>
+                    <div class="works">
+                        <div class="html"> <a href="#">
+                                <p class="word319">html</p>
+                            </a></div>
+                        <div class="css"><a href="#">
+                                <p class="word21">css</p>
+                            </a></div>
+                        <div class="jv"> <a href="#">
+                                <p class="word0">javascript</p>
+                            </a></div>
+                    </div>
```

Comment:
Kindly change this to be a `ul` tag containing multiple `li` tags instead of a `div` tag containing multiple `div` tags for semantic purposes. Please do this for all projects.